Sources/DataPipeline/DataPipeline.swift

```diff
@@ -23,25 +23,6 @@ public class DataPipeline: ModuleTopLevelObject<DataPipelineInstance>, DataPipel
         super.init(moduleName: Self.moduleName)
     }
 
-    /**
-     Initialize the shared `instance` of `DataPipeline`.
-     Call this function when your app launches, before using `DataPipeline.shared`.
-     */
-    @discardableResult
-    public static func initialize(
-        writeKey: String,
-        configure configureHandler: ((inout DataPipelineConfigOptions) -> Void)? = nil
-    ) -> CustomerIOInstance {
```

Comment:
Removed this as we are now using `DataPipeline.initialize(moduleConfig: DataPipelineConfigOptions)`